java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/dao/JobIdGenerator.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 4Paradigm
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com._4paradigm.openmldb.taskmanager.dao;
+
+import com._4paradigm.openmldb.common.zk.ZKClient;
+import com._4paradigm.openmldb.common.zk.ZKConfig;
+import com._4paradigm.openmldb.taskmanager.config.TaskManagerConfig;
+
+import java.nio.charset.StandardCharsets;
+
+public class JobIdGenerator {
+    private static int jobId = 0;
+    private volatile static ZKClient zkClient;
+
+    private JobIdGenerator() {
+    }
+
+    public static int getUniqueJobID() throws Exception {
+        synchronized (JobIdGenerator.class) {
+            if (zkClient == null) {
+                zkClient = new ZKClient(ZKConfig.builder()
```

Comment:
We may initialize `zkClient` when startup so that we don't have to check `zkClient` for each request.

Reply:
You're right. I will add a TODO. If we support to initialize zkClient when start, we can remove these codes.